content/_data/authors/vandit1604.adoc

```diff
@@ -0,0 +1,12 @@
+---
+name: "Vandit Singh"
+github: vandit1604
+twitter: vandittwts
+---
+
+Vandit is a computer science engineering undergraduate student from India. 
+In July 2022, he began his Opensource journey with Jenkins. 
+He is interested in a variety of technologies, including DevOps, Web Development and building things. 
+He mostly communicates with computers using Java, C, and C++ and Golang. 
+He was selected as a Google Summer of Code 2023 (GSoC'23) student to work on the project 
+link:https://www.jenkins.io/projects/gsoc/2023/project-ideas/alternative-jenkinsio-build-tool/[Building Jenkins.io using Alternative Tools].
```

Comment:
```suggestion
link:/projects/gsoc/2023/project-ideas/alternative-jenkinsio-build-tool/[Building Jenkins.io using Alternative Tools].
```

Reply:
Also, we should link to the [project page](https://www.jenkins.io/projects/gsoc/2023/projects/alternative-jenkinsio-build-tool/) with contents to be added later instead of the project idea page.